THEPageControl/Source/THEPageControl.swift

```diff
@@ -27,15 +27,24 @@ public class PageControl: UIView {
 
     public struct Dot {
         public struct Style {
+
+            public enum Shape {
+                case circle
+                case square
+            }
+
+            public var shape: Shape
             public var radius: Float
             public var fillColor: UIColor
             public var strokeColor: UIColor
             public var strokeWidth: Float
 
-            public init(radius: Float,
+            public init(shape: Shape,
```

Comment:
@wiruzx I'd rather add this parameter to the very end of the list of parameters and make it .circle by default for backward compatibility. Can you do you it, please?

Reply:
Sure 👍